### PR TITLE
DOC: signal: Some additional documentation to go along with the SOS enhancement.

### PR DESCRIPTION
**Pull request by WarrenWeckesser**
Here's the plot generated by the example in the `sosfilt_zi` docstring:
scipy-signal-sosfilt_zi-1

### CODE REVIEW

**Comment by ewmoore** [comment]
+1, nice examples and a very nice plot.

**Comment by Eric89GXL** [comment]
Agreed, no changes from me +1 for merge

File: scipy/signal/filter_design.py [hunk 1]
```diff
@@ -762,6 +762,71 @@ def zpk2sos(z, p, k, pairing='nearest'):
            with the ``pairing == 'keep_odd'`` method.
 
     .. versionadded:: 0.16.0
+
+    Examples
+    --------
+
```
Comment (ewmoore) [inline scipy/signal/filter_design.py]: I think this should read "a system".

Comment (WarrenWeckesser) [inline scipy/signal/filter_design.py]: Yup.  Thanks.

**Comment by ewmoore** [comment]
Thanks!

**Pull request merged**